controllers.js

```diff
@@ -34,4 +39,22 @@ exports.getSpecificCommentsByArticleId = (req, res, next) => {
     .catch((err) => {
         next(err)
     })
+}
+
+exports.postCommentByArticleId = (req, res, next) => {
+    const commentBody = req.body.body;
+    const commentAuthor = req.body.username;
+    checkIfArticleIdExists(req.params.article_id)
+    .then(({article_id}) => {
+        addCommentByArticleId(article_id, commentBody, commentAuthor)
+        .then((returnVal) => {
+            return res.status(201).send({ comment: returnVal })
```

Comment:
Great choice of key to return the comment on, nice an descriptive for when we come to use this on the front end 👍